src/app/components/queue-v2/queues-v2.component.ts

```diff
@@ -0,0 +1,300 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { QueueInfo } from '@app/models/queue-info.model';
+import { finalize } from 'rxjs/operators';
+import { SchedulerService } from '@app/services/scheduler/scheduler.service';
+
+import { select } from "d3-selection";
+import * as d3hierarchy from "d3-hierarchy";
+import * as d3flextree from "d3-flextree";
+import * as d3zoom from "d3-zoom";
+import { transition } from 'd3-transition'; // we need to import transition even if we don't use it explicitly
+
+export interface TreeNode {
+  name: string;
+  children?: TreeNode[];
+  _children?: TreeNode[];
+}
+
+@Component({
+  selector: 'queues-v2-view',
+  templateUrl: './queues-v2.component.html',
+  styleUrls: ['./queues-v2.component.scss']
+})
+
+export class QueueV2Component implements OnInit {
+  rootQueue: QueueInfo | null = null;
+
+  constructor(
+    private scheduler: SchedulerService,
+    private spinner: NgxSpinnerService,
+    private router: Router
+  ) {}
+
+  ngOnInit() {
+    this.fetchSchedulerQueuesForPartition()
+  }
+
+  fetchSchedulerQueuesForPartition() {
+    let partitionName = 'defau';
```

Comment:
Should be 'default'.

Reply:
<img width="1385" alt="image" src="https://github.com/apache/yunikorn-web/assets/26764036/80cbac96-5787-4c1b-af72-0b35203e9368">